src/parser/jobs/smn/modules/index.js

```diff
@@ -1,11 +1,11 @@
 import {Interrupts} from 'parser/core/modules/Interrupts'
+import {PetTimeline} from 'parser/core/modules/PetTimeline'
```

Comment:
What's the point of having a SMN specific PetTimeline module if using core now?

Reply:
I.. umm.. don't remember changing that to use core. (maybe vs did it?)  Either way, I'll remove the apparently unused SMN specific one after work and double check that nothing breaks.

Reply:
If the local one is relevant just change the import here.